internal/backend/azure/config.go

```diff
@@ -73,6 +74,8 @@ func (cfg *Config) ApplyEnvironment(prefix string) {
 		cfg.AccountSAS = options.NewSecretString(os.Getenv(prefix + "AZURE_ACCOUNT_SAS"))
 	}
 
+	cfg.ForceCliCredential = os.Getenv(prefix+"AZURE_FORCE_CLI_CREDENTIAL") == "true"
```

Comment:
Please use `strconv.ParseBool` for parsing the environment variable. I'd prefer to set `ForceCliCredential` only if the environment variable evaluates to true. (Basically the boolean analogue to the string parameters are currently handled).

Reply:
Done.